### DISCUSSION TITLE
sleep function in character LCD

### DISCUSSION THREAD

**Discussion by gitso7** [rern/rAudio#771] [Fixed Issues]
I suppose the sleep function in the character LCD configuration should activate a sleep mode after 60 seconds.
But nothing happens after that time (nor any time later).
Pushing the orange sleep button there works well (although not to my desire*).
I guess this is a bug.

*sleeping should (if possible!) not only blank the screen but also switch of the background lightning of the LCD

**Comment by rern** [comment]
Try:
`wget https://github.com/rern/rAudio-1/raw/UPDATE/srv/http/bash/lcdcharconfig.py -P /srv/http/bash`

**Comment by gitso7** [comment]
I did and experienced the following:
- Running it produced the file «lcdcharconfig.py.1» in «/stv/http/bash» - I asked myself why the «.1» in the file name...
- After rebooting still no sleep function, at least not within some five minutes
- I copied «lcdcharconfig.py.1» to «lcdcharconfig.py» and set it to http:http 0755
- After rebooting again still no sleep function, at least not within some five minutes

So it looks like not "working"

**Comment by gitso7** [comment]
Just as a feedback
- I updated to 220120
- the sleep function (after 60s) doesn't work
- pushing the orange sleep button worked once. The dislay "woke up" when the next song started = looks good
- pushing the orange button again switched off the display for the fraction of a second but the it woke up again but now showing nothing but the song time. With the start of the next song all the info (artist, song, album, time) was shown.

**Comment by rern** [comment]
Fixed.

**Comment by gitso7** [comment]
works (almost) fine
- pushing the orange button "sleeps" the display at once - ok
- stopping or pausing  the playing sleeps the display  after 60 seconds
- while playing, the display is never put into sleep mode 

In my opinion it'd be nice (or logical) if:
- stopping or pausing playing sould sleeps the display  at once
- while playing, the display should be put into sleep mode after 60 seconds
